### PR TITLE
Feature/add description field to meeting model

### PR DESCRIPTION
**Pull request by timDeHof**
# Description

 

This PR adds an optional description field to the meeting model schema, which allows users to provide a detailed description of the meeting. This improves the information available for each team meeting and provides more context for participants.  

## Issue link

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Feature updates / changes
- [ ] Tests
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- migrate the databases and seed it
- run unit and e2e tests 
- check in swagger UI that the creating meetings endpoint shows a description field in the response.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] I have updated the change log

### CODE REVIEW

**Comment by cherylli** [comment]
Everything else works fine, all tests passed

File: src/sprints/dto/create-team-meeting.dto.ts [hunk 1]
```diff
@@ -10,6 +10,14 @@ export class CreateTeamMeetingDto {
     })
     title: string;
 
+    @IsString()
```
[the remark this answers is not in the document]
Comment (cherylli) [reply src/sprints/dto/create-team-meeting.dto.ts:13]: In `voyages/sprints/{sprintNumber}/teams/{teamId}/meetings` when description is not supplied

image

**Comment by timDeHof** [comment]
I have added `IsOptional()` to the dto

**Comment by cherylli** [comment]
@JoshuaHinman @curtwl

**Review by JoshuaHinman — APPROVED** [review]
E2e tests passed, swagger example looks good.  I had to go up to sprint 4 to create a meeting without a conflict, but it works as described - nice work!

**Pull request merged**